recipes/r-textrecipes/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set version = '0.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-textrecipes
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/textrecipes_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/textrecipes/textrecipes_{{ version }}.tar.gz
+  sha256: acd44dca806ec58c31a7abdff0517d3c65ddd0f85819da205454a6742f3bd5da
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+
+  host:
+    - r-base
+    - r-snowballc
+    - r-dplyr
+    - r-generics
+    - r-magrittr
+    - r-purrr
+    - r-recipes >=0.1.4
+    - r-rlang
+    - r-stopwords
+    - r-stringr
+    - r-text2vec
+    - r-tibble
+    - r-tokenizers
+  run:
+    - r-base
+    - r-snowballc
+    - r-dplyr
+    - r-generics
+    - r-magrittr
+    - r-purrr
+    - r-recipes >=0.1.4
+    - r-rlang
+    - r-stopwords
+    - r-stringr
+    - r-text2vec
+    - r-tibble
+    - r-tokenizers
+
+test:
+  commands:
+    - $R -e "library('textrecipes')"           # [not win]
+    - "\"%R%\" -e \"library('textrecipes')\""  # [win]
+
+about:
+  home: https://github.com/tidymodels/textrecipes
+  license: MIT
+  summary: Converting text to numerical features requires specifically created procedures, which
+    are implemented as steps according to the  'recipes' package. These steps allows
+    for tokenization, filtering, counting (tf and tfidf) and feature hashing.
+  license_family: MIT
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - philip-khor
```

Comment:
@philip-khor please let me know if you plan to continue to work on conda and R and we can add you the team :)